src/lib/containers/TotalQueryResults.tsx

```diff
@@ -224,9 +224,9 @@ const TotalQueryResults: FunctionComponent<TotalQueryResultsProps> = ({
             el.columnName === columnName
               ? el.values.filter(el => el !== value)
               : el.values,
```

Comment:
We can't assume this is a ColumnSingleValueQueryFilter anymore!  Now, when a value is removed from an existing filter, use the original filter values (either going to have an operator if it's a single value filter, or function if it's a multi-value filter).